src/net/chat.rs

```diff
@@ -1182,11 +1232,28 @@ impl ConversationP2P {
         }
 
         let accept_data = HandshakeAcceptData::new(local_peer, self.heartbeat);
+        let stacks_message = if ConversationP2P::supports_stackerdb(local_peer.services)
+            && ConversationP2P::supports_stackerdb(self.peer_services)
+        {
+            StacksMessageType::StackerDBHandshakeAccept(
+                accept_data,
+                StackerDBHandshakeData {
+                    rc_consensus_hash: chain_view.rc_consensus_hash.clone(),
+                    // placeholder sbtc address for now
+                    smart_contracts: vec![
+                        ContractId::parse("SP000000000000000000002Q6VF78.sbtc").unwrap()
```

Comment:
Can you use a const for this placeholder?

Reply:
It's getting removed altogether in the follow-on #3552 PR.